src/mc_bench/migrations/versions/ae55800708f6_add_grok_model_and_provider.py

```diff
@@ -0,0 +1,56 @@
+"""add_grok_model_and_provider
+
+Revision ID: ae55800708f6
+Revises: d9dbc81fd67d
+Create Date: 2025-01-03 18:04:48.996163
+
+"""
+from typing import Sequence, Union
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision: str = 'ae55800708f6'
+down_revision: Union[str, None] = 'd9dbc81fd67d'
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None: # This I need to check a bit more 
+    op.execute("""\
+        INSERT INTO specification.provider_class (created_by, name) VALUES (
+            (select id from auth.user where username = 'SYSTEM'),
+            'GROK_SDK'
+        ) ON CONFLICT (name) DO NOTHING;
+
+        INSERT INTO specification.model (created_by, slug, active) VALUES (
```

Comment:
We haven't been including model and provider data in any migrations to the present.

We COULD, but we HAVEN'T been.

The intention is to add them manually via the UI. Don't feel strongly about this, it just hasn't been our convention.